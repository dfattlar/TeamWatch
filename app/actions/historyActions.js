'use strict';

import * as types from './actionTypes';

export function deleteHistory(id) {
    return {
        type: types.DELETE_HISTORY,
        id
    }
}
