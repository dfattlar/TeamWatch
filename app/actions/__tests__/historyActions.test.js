import * as actions from '../historyActions';

describe('History Actions', () => {
  it('should create an action to delete a history item', () => {
    const id = 'id_123';
    expect(actions.deleteHistory(id)).toMatchSnapshot();
  });
});
