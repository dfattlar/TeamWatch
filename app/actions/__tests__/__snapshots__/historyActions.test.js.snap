// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History Actions should create an action to delete a history item 1`] = `
Object {
  "id": "id_123",
  "type": "DELETE_HISTORY",
}
`;
